### ISSUE TITLE
[Aggregator] Aggregator stuck at batch 7355

### ISSUE THREAD

**Issue by mandrigin** [0xPolygon/cdk-erigon#619]
```
/src/aggregator/aggregator.go:284 github.com/0xPolygonHermez/zkevm-aggregator/aggregator.(*Aggregator).handleReceivedDataStream()
/go/pkg/mod/github.com/0x!polygon!hermez/zkevm-data-streamer@v0.2.2/datastreamer/streamclient.go:562 github.com/0xPolygonHermez/zkevm-data-streamer/datastreamer.(*StreamClient).getStreaming()
)       {"pid": 1, "version": "v0.0.2-RC26"}
github.com/0xPolygonHermez/zkevm-aggregator/aggregator.(*Aggregator).handleReceivedDataStream
        /src/aggregator/aggregator.go:284
github.com/0xPolygonHermez/zkevm-data-streamer/datastreamer.(*StreamClient).getStreaming
        /go/pkg/mod/github.com/0x!polygon!hermez/zkevm-data-streamer@v0.2.2/datastreamer/streamclient.go:562
2024-06-13T12:58:01.803Z        FATAL   datastreamer/streamclient.go:564172.20.0.13:34870 Processing entry 184566: no rows in result set. HALTED!       {"pid": 1, "version": "v0.1.0"}
github.com/0xPolygonHermez/zkevm-data-streamer/datastreamer.(*StreamClient).getStreaming
        /go/pkg/mod/github.com/0x!polygon!hermez/zkevm-data-streamer@v0.2.2/datastreamer/streamclient.go:564
```

**Comment by mandrigin** [comment]
should be fixed by #646

**Comment by mandrigin** [comment]
network 5 is fully proven, the issue is solved
